### PR TITLE
Update v1.1.0

### PR DESCRIPTION
**Pull request by rafaelmaeuer**
- feat: add helpers package (with `sleep` function)
  - feat: add numbers package (with `toReadable` function)
  - feat: add `capitalize` and `pluralize` functions in strings package
  - chore: move `isEven` from strings to numbers package (BREAKING CHANGE)

resolves most requests from https://github.com/swrlab/node-utils/issues/466

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ by [**SWR Audio Lab**](https://lab.swr.de/)
 
 ## Changelog
 
+- 2022-12-19 - v1.1.0
+  - feat: add helpers package (with `sleep` function)
+  - feat: add numbers package (with `toReadable` function)
+  - feat: add `capitalize` and `pluralize` functions in strings package
+  - chore: move `isEven` from strings to numbers package (BREAKING CHANGE)
```
Comment (frytg) [inline CHANGELOG.md:11]: I think long term we should match the Changelog-style with our other projects

File: packages/strings/README.md [hunk 2]
```diff
@@ -271,6 +269,32 @@ notNullOrUndefined(undefined)
...
+Import the library:
+
+```js
+const { pluralize } = require('@swrlab/utils/packages/strings')
+```
+
+Then use the toolkit:
+
+```js
+pluralize( 1, 'Apple' )
```
Comment (frytg) [inline packages/strings/README.md:285]: I guess Lint would critize the space before `1` and in the end

File: utils/strings/capitalize.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// capitalize value to Value
+module.exports = (value) => value[0].toUpperCase() + value.slice(1)
```
Comment (frytg) [inline utils/strings/capitalize.js:2]: Weird idea, but does that work with short strings like `a`, or would `.slice(1)` return `undefined`? Maybe we could also use the `${...}${...}` format (https://eslint.org/docs/latest/rules/prefer-template).

**Review by frytg — CHANGES_REQUESTED** [review]
I think we should also add the new sub-packages in the root README https://github.com/swrlab/node-utils/tree/dev/update#packages

File: CHANGELOG.md (hunk 1 above)
Comment (rafaelmaeuer) [reply CHANGELOG.md:11]: what do you mean? please specify or suggest changes

File: utils/strings/capitalize.js (hunk 3 above)
Comment (rafaelmaeuer) [reply utils/strings/capitalize.js:2]: this works: `capitalize("a") = A`, added to tests

**Review by frytg — APPROVED** [review]

**Pull request merged**